src/code.cloudfoundry.org/cf-tcp-router/watcher/watcher_test.go

```diff
@@ -84,18 +84,8 @@ var _ = Describe("Watcher", func() {
 
 		BeforeEach(func() {
 			tcpEvent = routing_api.TcpEvent{
-				TcpRouteMapping: models.NewTcpRouteMapping(
-					routerGroupGuid,
-					61000,
-					"some-ip-1",
-					5222,
-					0,
-					"",
-					nil,
-					0,
-					models.ModificationTag{},
-				),
-				Action: "Upsert",
+				TcpRouteMapping: models.NewTcpRouteMapping(routerGroupGuid, 61000, "some-ip-1", 5222, 0, "", nil, 0, models.ModificationTag{}, false, nil),
+				Action:          "Upsert",
```

Comment:
In many places you have changed our standard for breaking long function calls onto multiple lines. Please put them back into the format they were before.